pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockedPulsarServiceBaseTest.java

```diff
@@ -615,13 +615,12 @@ public static void deleteNamespaceWithRetry(String ns, boolean force, PulsarAdmi
      */
     public static void deleteNamespaceWithRetry(String ns, boolean force, PulsarAdmin admin,
                                                 Collection<PulsarService> pulsars) throws Exception {
-        Awaitility.await().atMost(5, TimeUnit.SECONDS).until(() -> {
+        Awaitility.await().ignoreExceptions().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
```

Comment:
> don't swallow the last exception

From the wiki page linked, it says only the exception will be evaluated to `false`. How do you expose the exception with this method call differs from what stacktrace we get currently?

Reply:
you are right. I'll add explicit logging of exceptions. My assumption is that it's the `PulsarAdminException.NotFoundException` which caused this.

Reply:
I added a minimum backoff of 500 milliseconds and added explicit logging. @tisonkun PTAL